wolfcrypt/src/port/caam/caam_driver.c

```diff
@@ -19,6 +19,11 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 
+#ifdef HAVE_CONFIG_H
+    #include <config.h>
+#endif
+
+#include <wolfssl/wolfcrypt/settings.h>
```

Comment:
Was avoiding adding wolfSSL header file includes into the driver (caam_*.c files). In my mind it increases the difficulty of building the driver separately and installing it on a kernel and of the eventual separation of the driver from wolfSSL if the driver capability grows and the code is large enough to merit being separated out of the wolfssl directory.

Reply:
We need a way to guard these files from compilers including all .c. The Keil CMSIS pack is currently broken because it tries to build these files. Can we at least add the QNX/Integrity compile option? If not we need a build macro to resolve this.

Reply:
Surrounding the files with QNX/Integrity compile options would be good. Having the same thing like caam_driver.c

```
  #if (defined(__INTEGRITY) || defined(INTEGRITY)) || \                           
      (defined(__QNX__) || defined(__QNXNTO__)) 
```